enterprise/coderd/workspaceagents_test.go

```diff
@@ -83,6 +84,10 @@ func TestBlockNonBrowser(t *testing.T) {
 func TestReinitializeAgent(t *testing.T) {
 	t.Parallel()
 
+	if runtime.GOOS == "windows" {
+		t.Skip("this test doesn't pass on Windows with Postgres")
```

Comment:
Please open an internal ticket

Reply:
the skip has already been merged separately: https://github.com/coder/coder/pull/17968